Implemented support for selecting multiple packages for installation

igor84 opened the pull request:
<img width="569" alt="image" src="https://github.com/GlitchEnzo/NuGetForUnity/assets/5306699/6dc45194-d84f-421f-8f30-804d0c84da25">

<img width="571" alt="image" src="https://github.com/GlitchEnzo/NuGetForUnity/assets/5306699/d72ad4de-e181-43a6-bf62-8d694954ceeb">

JoC0de commented on src/NuGetForUnity/Editor/Ui/NugetWindow.cs:
```diff
@@ -699,6 +769,48 @@ private void DrawOnline()
+
+            EditorGUI.LabelField(rectangle, string.Empty, headerStyle);
+
+            var arrow = showPackagesToInstall ? "\u2227" : "\u2228";
```
I would have used some constants for the string so we don't have "magic numbers"

JoC0de reviewed (approved):
sorry for the delay
looks good to me 👍
only one small note

igor84 commented:
I realized I didn't properly handle packages that are already installed or are available in Unity in the selection code so I fixed that. I also added a "Select all from clipboard" button so users can copy a list of package ids they need to install from somewhere and just click this button to have them all selected for installation.
